client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -445,6 +446,12 @@ export class CheckoutThankYou extends React.Component {
 
 			return (
 				<Main className="checkout-thank-you">
+					{ this.props.transferComplete && (
+						<WpAdminAutoLogin
+							site={ { URL: `https://${ this.props.site?.wpcom_url }` } }
```

Comment:
Firefox doesn't allow the auto-login with the wpcomstaging domain (possibly a cross-site rule), so I'm using the site's wordpress.com domain here.